ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayOverviewMap/DisplayOverviewMap.cpp

```diff
@@ -0,0 +1,81 @@
+// [WriteFile Name=DisplayOverviewMap, Category=Maps]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "DisplayOverviewMap.h"
+
+#include "FeatureLayer.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "ServiceFeatureTable.h"
+#include "Viewpoint.h"
+
+namespace
+{
+  QUrl featureTableURL = QUrl("https://services6.arcgis.com/Do88DoK2xjTUCXd1/arcgis/rest/services/OSM_Tourism_NA/FeatureServer/0");
+}
+
+using namespace Esri::ArcGISRuntime;
+
+DisplayOverviewMap::DisplayOverviewMap(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_basemap(new Basemap(BasemapStyle::ArcGISTopographic, this)),
```

Comment:
doesn't look to me like this needs to be a member. can you just initalize it inline when you create `m_map`?

m_map(new Map(new Basemap(BasemapStyle::ArcGISTopographic, this), this))